sass/variables/_colors.scss

```diff
@@ -92,6 +92,7 @@ $text-highlight: $lime-dark;
 $base-link-color: $sky-blue;
 $hover-link-color: $sky-blue-light;
 $visited-link-color: $purple;
+$alt-link-color: $anchor-blue;
```

Comment:
I was going to go with `subnav-text-color`, but the mobile nav also uses this color so I will use `nav-text-color`.  When we revisit this for the refactor story, I agree that revising some of the variables used in the main header will benefit us.